bundles/binding/org.openhab.binding.autelis/src/main/java/org/openhab/binding/autelis/internal/AutelisGenericBindingProvider.java

```diff
@@ -80,13 +76,8 @@ public void processBindingConfiguration(String context, Item item, String bindin
 		AutelisBindingConfig config = new AutelisBindingConfig(bindingConfig);
 		
 		addBindingConfig(item, config);	
-		
-		Set<Item> items = contextMap.get(context);
-		if (items == null) {
-			items = new HashSet<Item>();
-			contextMap.put(context, items);
-		}
-		items.add(item);
+
+		super.processBindingConfiguration(context, item, bindingConfig);
 	}
```

Comment:
Removed code duplicated what super.processBindingConfiguration had already done, so moved super call to end.